brayns/common/types.h

```diff
@@ -373,6 +373,14 @@ struct RpcParameterDescription : RpcDescription
     std::string paramName;
     std::string paramDescription;
 };
+
+enum class BVHType
+{
+    dynamic,
```

Comment:
that's very ospray centric. what is 'none' even? Shouldn't it be a bitmask also? dynamic + compact sounds reasonable to me. If ospray can't do that, we shouldn't limit it here. 'Static' is missing also to me.

Reply:
Since optimizations for a particular metric typically go against others, I think it's fair to have a single label to provide a hint about the expected usage.